clang/test/CodeGen/aarch64-fmv-dependencies.c

```diff
@@ -0,0 +1,381 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --check-attributes --check-globals all --filter "define|attributes" --include-generated-funcs --version 4
+
+// Test/document all of the dependencies between possible AArch64 FMV extensions.
+// Also test the name mangling.
+
+// RUN: %clang --target=aarch64-linux-gnu --rtlib=compiler-rt -emit-llvm -S -o - %s | FileCheck %s
+
+
+int __attribute__((target_version("aes"))) fmv(void) { return 0; }
```

Comment:
nit: I haven't mentioned this before, but I find `<return type> <attribute> <name>(<args>) { <body> }` to be a really confusing order for these.

Why not: `<attribute> <return type> <name>(<args>) { <body> }` ?

Reply:
I agree, done.